.github/workflows/test.yml

```
@@ -0,0 +1,34 @@
+name: Test collection
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  sanity:
+    uses: ansible-network/github_actions/.github/workflows/sanity.yml@main
+  unit-galaxy:
+    uses: ansible-network/github_actions/.github/workflows/unit_simple.yml@main
+  unit-source:
+    uses: ansible-network/github_actions/.github/workflows/unit_simple.yml@main
+    with:
+      collection_pre_install: >-
+        git+https://github.com/ansible-collections/ansible.utils.git
+        git+https://github.com/ansible-collections/ansible.netcommon.git
+      matrix_exclude: >-
+        [
+          {
+            "ansible-version": "stable-2.9",
+            "python-version": "3.8"
+          },
+          {
+            "ansible-version": "stable-2.9",
+            "python-version": "3.9"
+          },
+          {
+            "ansible-version": "stable-2.9",
+            "python-version": "3.10"
```

I'm _pretty_ sure this should be sufficient as we are exclufing 2.9 across all Python versions
```suggestion
            "ansible-version": "stable-2.9",
```

---

trying it out